### PR TITLE
Added api docs for delete backend service.

### PR DESCRIPTION
**Pull request by y0geshdev**

### CODE REVIEW

**Review by m-orfanos — APPROVED** [review]

File: source/includes/gcp/_backend_services.md [hunk 1]
```diff
@@ -1 +1,15 @@
...
+#### Backend services
+
+<!-------------------- DELETE A BACKEND SERVICE -------------------->
+
+##### Delete a backend service
+
+```shell
+curl -X DELETE \
+   -H "MC-Api-Key: your_api_key" \
+   "https://cloudmc_endpoint/v1/services/gcpservicecode/gcpenvironment/backendservices/2570199154720991429"
```
Comment (SeventhOfNine) [inline source/includes/gcp/_backend_services.md]: Can you change the URL to match the rest of the documentation:

`https://cloudmc_endpoint/v1/services/gcp/test-area/backendservices/2570199154720991429`

**Review by SeventhOfNine — CHANGES_REQUESTED** [review]
Small change for the URL

File: source/includes/gcp/_backend_services.md (hunk 1 above)
Comment (y0geshdev) [reply source/includes/gcp/_backend_services.md]: Updated the url to keep it consistent with rest of the api doc.

**Comment by y0geshdev** [comment]
> Small change for the URL

Made the url changes.

**Review by SeventhOfNine — APPROVED** [review]
Looks good, thanks for the changes.

**Pull request merged**